opentelemetry-sdk/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## vNext
-
+- Replace TestExporter with InMemoryExporter [#1624](https://github.com/open-telemetry/opentelemetry-rust/pull/1624)
```

Comment:
please remove this, we dont need changelog for internal testing changes. only for end users facing changes, we need to make entry in here. Thanks.